CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ endif()
 set_target_properties(v4l2sink PROPERTIES PREFIX "")
 
 install(TARGETS v4l2sink
-	LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/obs-plugins)
+	LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/x86_64-linux-gnu/obs-plugins)
```

Comment:
This looks incorrect as this hardcode the libdir for x86_64 and is a debianism. (others distro don't use this scheme).

Usually CMAKE_INSTALL_LIBDIR needs to be configured as appropriate. Or this should be inherited from FindLibObs.cmake (from the obs-studio devel package)...

Reply:
(and a project maintainer).